crit/mempages.go

```diff
@@ -0,0 +1,145 @@
+package crit
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"syscall"
+
+	"github.com/checkpoint-restore/go-criu/v6/crit/images/pagemap"
+)
+
+var sysPageSize = uint64(syscall.Getpagesize())
```

Comment:
Any specific reason for not using `os.Getpagesize()`. Looking at the Go sources it just calls `syscall.Getpagesize()`, but I think using `os` would be better.